src/Amp/Application.php

```diff
@@ -108,7 +108,7 @@ public function loadContainer() {
       $container->setParameter('instances_timeout', getenv('AMP_INSTANCES_TIMEOUT'));
     }
 
-    $container->setAlias('mysql', 'mysql.' . $container->getParameter('mysql_type'));
+    $container->setAlias('db', 'db.' . $container->getParameter('db_type'));
```

Comment:
There is one blocker issue in the design. Renaming ```mysql_type``` to ```db_type``` breaks existing installations -- eg if an installation currently uses ```mysql_type==mycnf``` and then runs the new code, it will effectively revert to the system default (```db_type==mysql_dsn```) -- but it should really become ```db_type==mysql_mycnf```.

I suspect the simplest way to fix this would be to add more nuance in calculating ```db```. Maybe this: in ```services.yml```, remove the default value of ```db_type```; then, in ```Application.php```,

```
if ($container->getParameter('db_type')) {
  $container->setAlias('db', ...);
} elseif ($container->getParameter('mysql_type')) {
  $container->setAlias('db', ...);
} else {
  $container->setAlais('db', 'mysql_dsn'); // default
}
```